metricbeat/module/elasticsearch/node/node_test.go

```diff
@@ -48,7 +48,6 @@ func TestFetch(t *testing.T) {
 				w.WriteHeader(200)
 				w.Header().Set("Content-Type", "application/json;")
 				w.Write([]byte(response))
-				assert.Equal(t, "/_nodes/_local", r.RequestURI)
```

Comment:
Is this not correct anymore?

Reply:
That it works locally and fails on CI makes me worry now because I think it passed previously.

Reply:
Alright, digging into it some more...